### PR TITLE
[ED27] [ED63] [ED53] - Import Partners/Products/Chart of Accounts + Accounting configuration of all of them + Banking Information

### PR DESCRIPTION
**Pull request by nassym-xevlabs**
- Added all the files needed to import partners, products and chart of accounts (as well as their accounting configuration)
- Implemented the Banking Info configuration (attached to corrected companies and Partners)

Warning: on my local setup, the Axelor demo data import for accounting and base modules took approx. 2hours+ and more than 100Gb in temp files (I don't know it the fact that I have the TRACE config for logs plays a role but I prefer to let you all know). So be sure to have an appropriate setup before testing it (server, database instance, etc.) to avoid any issue.

### CODE REVIEW

**Comment by nassym-xevlabs** [comment]
Don't know why 2 Menu.xml are changed as they were not edited and pulled from main branch to create that one.

**Comment by AntoineXev** [comment]
@nassym-xevlabs could you fix the issue related to the lang folders on this one ?

**Comment by nassym-xevlabs** [comment]
Everything has been updated and all the fixes implemented, including the deletion of the EN folders in base and accounting modules.

All the changes have been committed and pushed to the branch.

File: axelor-account/src/main/resources/demo/account-config.xml [hunk 1]
```diff
@@ -184,11 +205,56 @@
...
+  <!--<input file="account_accountManagement.csv" separator=";" type="com.axelor.apps.account.db.AccountManagement"
+    search="self.importId = :importId"> <bind search="self.importId = :company_id" to="company" update="true"/>
+    <bind search="self.code = :cashAccount_code and self.company.importId = :company_id" to="cashAccount"
+    update="true" if="cashAccount_code"/> <bind search="self.code = :saleAccount_code and self.company.importId
+    = :company_id" to="saleAccount" update="true" if="saleAccount_code"/> <bind search="self.code = :purchaseAccount_code
+    and self.company.importId = :company_id" to="purchaseAccount" update="true" if="purchaseAccount_code"/>
+    <bind search="self.importId = :sequence_importId and self.company.importId = :company_id" to="sequence"
+    update="true" if="sequence_importId"/> <bind search="self.importId = :journal_importId and self.company.importId
+    = :company_id" to="journal" update="true" if="journal_importId"/> <bind search="self.importId = :bankDetails_importId"
+    to="bankDetails" update="true" if="bankDetails_importId"/> </input> -->
```
Comment (AntoineXev) [inline axelor-account/src/main/resources/demo/account-config.xml]: Code could be removed (do not let comment that are not real ones in the code)

File: axelor-account/src/main/resources/demo/account-config.xml [hunk 2]
```diff
@@ -184,11 +205,56 @@
...
+
+    <bind search="self.bankDetails.importId = :bankDetails_importId" to="bankDetails"
+      if="bankDetails_importId"/>
+
+  </input>
+
+
+  <!--<input file="base_bankDetails.csv" separator=";" type="com.axelor.apps.base.db.BankDetails" search="self.company.importId
+    = :importId" update="true"> <bind to="company" search="self.importId = :importId"/> <bind to="partner"
+    search="self.importId = :partner_importId"/> </input> -->
```
Comment (AntoineXev) [inline axelor-account/src/main/resources/demo/account-config.xml]: Same here

File: axelor-base/src/main/resources/demo/base-config.xml [hunk 3]
```diff
@@ -151,11 +176,12 @@
...
-    <bind to="partner" column="partner_importId" search="self.importId =:partner_importId"
-      update="true"/>
-    <bind to="isDefault" eval="true"/>
-  </input>
+
+
+  <!--<input file="base_bankDetails.csv" separator=";" type="com.axelor.apps.base.db.BankDetails" search="self.company.importId
+    = :importId"> <bind to="company" search="self.importId = :importId"/> <bind to="partner" search="self.importId
+    = :partner_importId"/> </input> -->
+
```
Comment (AntoineXev) [inline axelor-base/src/main/resources/demo/base-config.xml]: Same

File: axelor-base/src/main/resources/demo/base-config.xml [hunk 4]
```diff
@@ -186,18 +212,55 @@
...
+    <bind to="name" column="name"/>
+    <bind to="code" column="code"/>
+
+
+    <!-- unit -->
+    <!--<bind to="name" column="name"/> -->
+
+
+
+
```
Comment (AntoineXev) [inline axelor-base/src/main/resources/demo/base-config.xml]: Too many spaces

File: axelor-base/src/main/resources/demo/base-config.xml [hunk 5]
```diff
@@ -285,22 +348,16 @@
...
+  <!--<input file="account_accountManagement.csv" separator=";" type="com.axelor.apps.account.db.AccountManagement"
+    search="self.importId = :importId"> <bind search="self.importId = :company_id" to="company" update="true"/>
+    <bind search="self.code = :cashAccount_code and self.company.importId = :company_id" to="cashAccount"
+    update="true" if="cashAccount_code"/> <bind search="self.code = :saleAccount_code and self.company.importId
+    = :company_id" to="saleAccount" update="true" if="saleAccount_code"/> <bind search="self.code = :purchaseAccount_code
+    and self.company.importId = :company_id" to="purchaseAccount" update="true" if="purchaseAccount_code"/>
+    <bind search="self.importId = :sequence_importId and self.company.importId = :company_id" to="sequence"
+    update="true" if="sequence_importId"/> <bind search="self.importId = :journal_importId and self.company.importId
+    = :company_id" to="journal" update="true" if="journal_importId"/> <bind search="self.importId = :bankDetails_importId"
+    to="bankDetails" update="true" if="bankDetails_importId"/> </input> -->
```
Comment (AntoineXev) [inline axelor-base/src/main/resources/demo/base-config.xml]: same

**Review by AntoineXev — CHANGES_REQUESTED** [review]
Needs Formating and comments, but rest looks good !

**Review by AntoineXev — APPROVED** [review]

**Pull request merged**